cmd/ctl/pkg/status/certificate/types.go

```diff
@@ -70,6 +73,9 @@ type IssuerStatus struct {
 	Kind string
 	// Conditions of Issuer/ClusterIssuer resource
 	Conditions []cmapiv1alpha2.IssuerCondition
+	// boolean indicating if Issuer/ClusterIssuer is a ACME Issuer/ClusterIssuer
+	// Defaults to false even Error is not nil
```

Comment:
`Defaults to false even if Error is not nil`, does that help?

Reply:
I guess I don't understand how that's any different to `Defaults to false`. I don't know why anyone would think
it would be `true` if there was an error. It probably doesn't hurt to cover that case though.